php_fpm/README.md

```diff
@@ -72,19 +97,16 @@ server {
 
 If you find this approach too tedious at scale, setting `use_fastcgi` to `true` instructs the check to bypass any proxy servers and communicate directly with PHP-FPM using FastCGI. The default port is `9000` for when omitted from `status_url` or `ping_url`.
 
-### Unix sockets
+##### Unix sockets
 
-If your php-fpm installation uses unix sockets, you have to use the below syntax for `status_url`, `ping_url` and enable `use_fastcgi`:
+If your PHP-FPM installation uses unix sockets, you have to use the below syntax for `status_url`, `ping_url` and enable `use_fastcgi`:
```

Comment:
unix sockets won't play nice with autodiscovery if the agent runs in a separate container/task/pod, as it won't have access to the file. It may be worth mentioning. If we have a page that explains running the agent as a sidecar maybe let's point to it, too.

Reply:
Doc updated